src/Message.php

```diff
@@ -1,6 +1,7 @@
 <?php
 namespace sngrl\PhpFirebaseCloudMessaging;
 
+use ReturnTypeWillChange;
```

Comment:
By adding #[\ReturnTypeWillChange] you can delete this.

Reply:
Complete